docker/build.sh

```diff
@@ -1,7 +1,7 @@
 # this script build the ocelot docker file
-version=0.0.9
-docker build --platform linux/amd64 -t mijitt0m/ocelot-build -f Dockerfile.base .
+version=0.0.10
+docker build --platform linux/amd64 -t ggnaegi/ocelot-build -f Dockerfile.base .
 echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
-docker tag mijitt0m/ocelot-build mijitt0m/ocelot-build:$version
-docker push mijitt0m/ocelot-build:latest
-docker push mijitt0m/ocelot-build:$version
\ No newline at end of file
+docker tag ggnaegi/ocelot-build ggnaegi/ocelot-build:$version
```

Comment:
Pay attention to the env vars. In CircleCI env the vars belong to Tom, right?
It seems we cannot change them. And do these vars work for your personal Docker account?
If Not, we have to rollback to Tom's account...
Going to check this...

Reply:
Yes, but we need to switch to the new docker repo you have foreseen

Reply:
Yes. ocelot2

Reply:
Ok, ready for try and error...